### PR TITLE
Add bahasa indonesia translation

### PR DESCRIPTION
**Pull request by jateez**
This PR addresses #279.

I added support for Bahasa Indonesia, hoping this helps more Indonesian speakers use Peri!

### CODE REVIEW

File: src/utils/translations/id.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
+const id = {
+  // Home Tab
+  Home: "Beranda",
+  "Period in": "Haid akan dimulai dalam",
+  "no info": "tidak ada informasi",
+  Period: "Haid",
+  today: "hari ini",
+  Delay: "Terlambat",
+  Days: "(0)[Hari];(1)[Hari];(2-4)[Hari];(5-20)[Hari];(21)[Hari];(22-24)[Hari];(25-30)[Hari];(31)[Hari];(32-34)[Hari];(35-40)[Hari];",
```
Comment (imblowfish) [inline src/utils/translations/id.ts:9]: You are writing `Hari` in any case, so you just can write `(0-inf)[Hari]` instead

**Review by imblowfish — CHANGES_REQUESTED** [review]

File: src/utils/translations/id.ts (hunk 1 above)
Comment (jateez) [reply src/utils/translations/id.ts]: okay!

**Review by imblowfish — APPROVED** [review]

**Comment by IraSoro** [comment]
Hello, @jateez! Thanks for your interest in the project!

Could you please answer the question about translation?
You translated the phrase "Period in 5 days" as "Menstruasi akan dimulai dalam 10 hari". I used Google Translate. It showed the translation of this phrase as "Period will start in 10 days."  Would it be correct to translate phrase "Period in 5 days" as "Menstruasi dalam 10 hari"?

**Comment by jateez** [comment]
Hi, sorry for the slow response, I was busy with my exams. 

The literal translation of "Period in 10 days" is "Menstruasi dalam 10 hari", so yes that's correct.  I chose to translate it as "Menstruasi akan dimulai dalam 10 hari" because I felt it would be easier for Indonesian speakers to understand.

**Comment by IraSoro** [comment]
> Hi, sorry for the slow response, I was busy with my exams.
> 
> The literal translation of "Period in 10 days" is "Menstruasi dalam 10 hari", so yes that's correct. I chose to translate it as "Menstruasi akan dimulai dalam 10 hari" because I felt it would be easier for Indonesian speakers to understand.

@jateez, thanks for the answer! We just want the translation string to have a fixed length if the language allows it. If we use the second option it would look like this
{AE5C51B0-973F-40AD-AB1C-696A3F8C9974}
But if you think the first option is better for understanding, we can leave this option and, for example, fix the text size in the future. This option looks like this:
{3BECF8A6-8C81-4A11-9838-926D1D2E3891}

**Comment by jateez** [comment]
I'm sure a native Indonesian speaker will still easily understand the "Haid dalam ..." options. However, those non-native speakers or those still learning Bahasa Indonesia might find it confusing at first but they'll likely understand it eventually. That said, I believe they would prefer to use the app in english instead.

I think if you prefer to go with the "Haid dalam ..." options, that is fine. @IraSoro

**Review by IraSoro — APPROVED** [review]

**Review by imblowfish — APPROVED** [review]

**Comment by IraSoro** [comment]
@jateez, thanks for translating into Indonesian! And congratulations on your first contribution to the `peri` project!

Good luck on your exams!

**Comment by jateez** [comment]
thank you!

**Pull request merged**